area/area.go

```diff
@@ -0,0 +1,34 @@
+package main
```

Comment:
Consider an input that represents sides `1 1 2`. Such triangle doesn't exist. What will your program output?